### DISCUSSION TITLE
Editing a command using the default editor

### DISCUSSION THREAD

**Discussion by JosephNuthalapati2244** [nushell/nushell#6792] [Q&A] [answered]
In bash, the shortcut `Ctrl+x Ctrl+e` can open the current shell command in an editor. On saving, the edited command is inserted back at the command prompt.

What is nushell equivalent of this?

**Comment by fdncred** [comment, marked as answer]
i believe it defaults to ctrl+o in nushell. you can see this with the command `keybindings default`
